brainwave-prediction-app/gui_windows/signal_module.py

```diff
@@ -0,0 +1,44 @@
+import time
+import threading
+
+
+class signalling_system:
+    """
+    This class intends to send the drone a non-movement signal every 10 seconds
```

Comment:
Could you update the comment here and on line 16 to "2 seconds", to ensure future users don't think the sleep timer was misentered.

Reply:
Will do that, thanks for spotting that.